.github/workflows/vm-pr.yml

```diff
@@ -93,6 +93,8 @@ jobs:
       matrix:
         fork: [
           'Berlin',
+          'MuirGlacier',
+          'Istanbul',
```

Comment:
Please always explicitly note if you add HFs to the fork matrix, this is a significant change which should not happen implicitly, since this might overload CI usage at some point. I guess it's ok here regarding the recent performance gains, still a bit unsure though if we are running too much on the normal PR runs.

Reply:
Just checked, this is only in the extended tests, so it only is triggered if you put the test all hardforks label on the PR 😄  So this should not have impact. Will nevertheless note in the future.

Reply:
Ah ok, thanks, wasn't aware, then it is less problematic.